docs/build/smart-contracts/learn-cadence.md

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 1
+slug: /build/learn-cadence
+title: Learn Cadence ↗️
```

Comment:
add `redirect` front matters

Reply:
@nialexsan that didn't work.  The router treated it as a relative link and was constructing the link as `https://docs.cadence.org/https://cadence-lang.org/docs/`

Is there a way to tell the router in the frontmatter that it's an outbound link?